src/main/common/filter.c

```diff
@@ -273,3 +273,103 @@ FAST_CODE float ptnFilterApply(ptnFilter_t *filter, float input) {
 
 	  return filter->state[filter->order];
 } // ptnFilterApply
+
+void luluFilterInit(luluFilter_t *filter, int N)
```

Comment:
I would create a new file dedicated to this filter.

Reply:
Will do!